src/modelfree/single_score_agent.py

```diff
@@ -0,0 +1,63 @@
+import gym
```

Comment:
No! :( This can definitely be  supported within score_agent. Please avoid this level of code duplication. @MichaelDDennis